include/gmsh.tcc

```diff
@@ -109,23 +111,143 @@ void GMSH::read_elements(const std::string& filename) {
         istream >> elementry;
 
         //! \brief Check element type
-        //! \details If element type not == to Tdim, skip element
-        if (elementtype != ndimension) {
+        //! \details If element type not == to specified elementtype_, skip
+        //! element
+        if (elementtype != elementype) {
```

Comment:
Ah yes, I removed the trailing _ in previous commit, but should rename it to something else.